src/api/routes/articleRouter.js

```diff
@@ -0,0 +1,18 @@
+import { Router } from 'express';
+import articleController from '../controllers/articleController';
+import checkArticle from '../../middlewares/getOneArticle';
+// import checkVerification from '../../middlewares/checkUserVerification';
+import checkArticleOwner from '../../middlewares/checkArticleOwnership';
+import checkValidToken from '../../middlewares/checkValidToken';
+
+const articleRouter = new Router();
+
+articleRouter.put('/:slug',
+  checkValidToken,
+  // checkVerification.checkUserExistance,
```

Comment:
remove this commented line please @MCFrank16

Reply:
I commented it cause I think we need it, just to check if the user is verified, I think we will not allow an unverified user to update the article or whatever else